From c312a2dac31981d0096d4066282573335df11c6d Mon Sep 17 00:00:00 2001
From: Haihua Hu <b55597@freescale.com>
Date: Tue, 8 Dec 2015 16:06:34 +0800
Subject: [PATCH 12/18] [glplugin] Fix glimagesink wayland resize showed
 blurred screen

For imx, wl_egl_window type is not a wl_proxy object. Can not set
queue to wl_egl_window object.

Upstream Status: Inappropriate [i.MX specific]

Signed-off-by: Haihua Hu <b55597@freescale.com>
Signed-off-by: Lyon Wang <lyon.wang@freescale.com>
---
 gst-libs/gst/gl/wayland/gstglwindow_wayland_egl.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/gst-libs/gst/gl/wayland/gstglwindow_wayland_egl.c b/gst-libs/gst/gl/wayland/gstglwindow_wayland_egl.c
index da35bf0..b20be22 100644
--- a/gst-libs/gst/gl/wayland/gstglwindow_wayland_egl.c
+++ b/gst-libs/gst/gl/wayland/gstglwindow_wayland_egl.c
@@ -325,9 +325,6 @@ create_surfaces (GstGLWindowWaylandEGL * window_egl)
 
     window_egl->window.native =
         wl_egl_window_create (window_egl->window.surface, width, height);
-    if (window_egl->window.surface_queue)
-      wl_proxy_set_queue ((struct wl_proxy *) window_egl->window.native,
-          window_egl->window.surface_queue);
   }
 }
 
-- 
1.9.1

